test/CMakeLists.txt

```diff
@@ -406,3 +406,11 @@ add_subdirectory(ci)
 if(WIN32)
   add_subdirectory(performance)
 endif()
+
+add_custom_target(
+  check-package
+  COMMAND ${CMAKE_COMMAND}
+    -DCMAKE_PREFIX_PATH="${CMAKE_BINARY_DIR}/../dist"
```

Comment:
Changed that variable into CMAKE_INSTALL_PREFIX. Is that OK?

Reply:
It will still not always work because you can install at an arbitrary location with `cmake --install --prefix`. 🤔

Reply:
Hmm, I can always read that path from ENV variable if that is needed like:
`TILEDB_INSTALL_DIR=<DIR> cmake --build <> --target check-package`

in CI/CD

Reply:
> It will still not always work because you can install at an arbitrary location with cmake --install --prefix. 🤔

Do we need to test this situation?

Reply:
IMHO not, we are not doing anything weird in our installed CMake config. The current arrangement with `CMAKE_INSTALL_PREFIX` might be good enough for CI scenarios.